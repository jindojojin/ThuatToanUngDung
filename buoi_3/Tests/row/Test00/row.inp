6
2 5 2
1 4 4
3 6 2
2 7 4
4 4 2
1 9 2
8
4 6 2
5 6 2
4 5 2
0 1 2
0 6 2
1 4 4
0 4 2
3 6 1
