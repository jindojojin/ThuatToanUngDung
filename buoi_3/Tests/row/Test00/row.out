25
9
16
10
53
18
28
0
